CRM/Export/BAO/ExportProcessor.php

```diff
@@ -1109,7 +1109,20 @@ public function getTransformedFieldValue($field, $iterationDAO, $fieldValue, $me
     ) {
       //check for custom data
       if ($cfID = CRM_Core_BAO_CustomField::getKeyID($field)) {
-        return CRM_Core_BAO_CustomField::displayValue($fieldValue, $cfID);
+        $html_type = CRM_Core_DAO::getFieldValue('CRM_Core_DAO_CustomField', $cfID, 'html_type');
+        $displayValue = CRM_Core_BAO_CustomField::displayValue($fieldValue, $cfID);
+        //need to calculate the link to the file for file custom data
+        if ($html_type === 'File') {
+          preg_match('/<a[^>]+href=([\'"])(?<href>.+?)\1[^>]*>/i', $displayValue, $matches);
+          if (!empty($matches)) {
+            $displayValue = $matches['2'];
+            if ((strpos($file, "http ")) === FALSE) {
+              return CIVICRM_UF_BASEURL . html_entity_decode($displayValue);
+            }
+          }
+        }
+
+        return $displayValue;
       }
 
       elseif (in_array($field, [
```

Comment:
Also this is weird that it gets thru the style checker. I guess a blank line before elseif is allowed?

Reply:
I guess, do you want me to remove this as well?

Reply:
If you're making other changes anyway then yes please!